internal/fsutil.go

```diff
@@ -72,6 +73,55 @@ func createTestExtImg(ctx context.Context, t *testing.T, extName string, testFil
 	return imgFile.Name()
 }
 
+// DeviceFile represents a block device
```

Comment:
DeviceFile should be renamed as BlockDeviceFile if it only used for block device.

Reply:
Will fix that. Do you have any idea regarding to the error returned by the integration test?
```
Messages:   	Can not mkfs.ext4 for the new block device: mke2fs 1.43.4 (31-Jan-2017)
 /tmp/208162464/example/hey is apparently in use by the system; will not make a filesystem here!
```

Reply:
Found the root cause, see comment below.